init.lua

```diff
@@ -268,73 +229,53 @@ minetest.register_on_joinplayer(function(player)
 		hangglider.id[pname] = player:hud_add({
 			hud_elem_type = "image",
 			text = "blank.png",
-			position = {x=0, y=0},
-			scale = {x=-100, y=-100},
-			alignment = {x=1, y=1},
-			offset = {x=0, y=0},
+			position = {x = 0, y = 0},
+			scale = {x = -100, y = -100},
+			alignment = {x = 1, y = 1},
+			offset = {x = 0, y = 0},
 			z_index = -150
 		})
 	end
 	if debug then
 		hangglider.debug[pname] = {id = player:hud_add({hud_elem_type = "text",
-			position = {x=0.5, y=0.1},
+			position = {x = 0.5, y= 0.1 },
 			text = "-",
 			number = 0xFF0000}),  -- red text
 			-- ht = {50,50,50},
 		}
 	end
-	--hangglider.airbreak[pname] = false
 end)
 
 minetest.register_on_leaveplayer(function(player)
 	local pname = player:get_player_name()
 	hangglider.use[pname] = nil
 	if HUD_Overlay then hangglider.id[pname] = nil end
 	if debug then hangglider.debug[pname] = nil end
-	--hangglider.airbreak[pname] = nil
 end)
 
 minetest.register_tool("hangglider:hangglider", {
 	description = "Glider",
 	inventory_image = "glider_item.png",
-	stack_max=1,
+	stack_max = 1,
 	on_use = function(itemstack, player)
 		if not player or player.is_fake_player then
 			-- player does not exist or is created from an automated machine (fake_player)
 			return
 		end
 		local pos = player:get_pos()
 		local pname = player:get_player_name()
-		if not hangglider.use[pname] then --Equip
-			minetest.sound_play("bedsheet", {pos=pos, max_hear_distance = 8, gain = 1.0})
+		if not hangglider.use[pname] then -- Equip
+			minetest.sound_play("bedsheet", {pos = pos, max_hear_distance = 8, gain = 1.0}, true)
 			if HUD_Overlay then player:hud_change(hangglider.id[pname], "text", "glider_struts.png") end
 			local airbreak = false
-			--[[if vel < -1.5 then  -- engage mid-air, falling fast, so stop but ramp velocity more quickly
-				--hangglider.airbreak[pname] = true
-				airbreak = true
-				local stopper = minetest.add_entity(pos, "hangglider:airstopper")
-				minetest.after(0, function(stopper, player) --"Extreme Measures"
-					stopper:set_pos(player:get_pos())
-					stopper:get_luaentity().attach = player
-					player:set_attach( stopper, "", {x=0,y=0,z=0}, {x=0,y=0,z=0})
-				end, stopper, player)
-			end]]
 			if not airbreak then
-				if moveModelUp then
-					minetest.add_entity(pos, "hangglider:glider"):set_attach(player, "", {x=0,y=10,z=0}, {x=0,y=0,z=0})
-				else
-					minetest.add_entity(pos, "hangglider:glider"):set_attach(player, "", {x=0,y=0,z=0}, {x=0,y=0,z=0})
-				end
+				minetest.add_entity(pos, "hangglider:glider"):set_attach(player, "", {x=0,y=10,z=0}, {x=0,y=0,z=0})
 			end
```

Comment:
To cleanup bit more just replace this whole block with single line:
```
minetest.add_entity(pos, "hangglider:glider"):set_attach(player, "", {x=0,y=10,z=0}, {x=0,y=0,z=0})
```

`airbreak` variable is never really used.